src/SchemaDesign/SchemaDesigner.tsx

```diff
@@ -0,0 +1,416 @@
+                              {step.questions.map((question, index) => (
+                                <Draggable key={question.title} draggableId={question.title} index={index}>
+                                  {(draggableProvided: DraggableProvided) => (
+                                    <div
+                                      ref={draggableProvided.innerRef}
+                                      {...draggableProvided.draggableProps}
+                                      {...draggableProvided.dragHandleProps}
+                                    >
+                                      <Box sx={{ m: 2 }}>
+                                        <Stack direction='row' alignItems='center' spacing={2}>
+                                          <Icon color='primary'>{getQuestionIcon(question.type)}</Icon>
+                                          <Typography key={question.title}>{question.title}</Typography>
+                                          <IconButton
+                                            onClick={() => deleteQuestion(question.reference)}
+                                            aria-label='delete'
+                                          >
+                                            <DeleteIcon />
+                                          </IconButton>
+                                        </Stack>
+                                      </Box>
+                                    </div>
+                                  )}
+                                </Draggable>
+                              ))}
+                              {droppableProvided.placeholder}
+                            </div>
+                          )}
+                        </Droppable>
+                      </DragDropContext>
+                      <Button onClick={handleClickOpen}>Add new question</Button>
+                    </Box>
+                  )}
+                </Box>
+              ))}
+              {selectedStep === '' && <Typography>Add a new step to begin designing your schema</Typography>}
+            </Paper>
+          </Box>
+        </Grid>
+        <Grid item xs={12} sm={12} md={6}>
+          <Typography sx={{ mb: 2 }} variant='h4'>
+            Preview
+          </Typography>
+          <Paper
+            sx={{
+              p: 2,
+              width: '100%',
+            }}
+          >
+            {showForm && splitSchema?.steps[stepIndex]?.schema === undefined && (
+              <Typography>Nothing to preview!</Typography>
+            )}
+            {showForm && splitSchema?.steps[stepIndex]?.schema !== undefined && (
+              <SchemaForm
+                schema={splitSchema.steps[stepIndex].schema}
+                formData={splitSchema.steps[stepIndex].state}
+                uiSchema={splitSchema.steps[stepIndex].uiSchema}
+              >
+                {/* eslint-disable-next-line react/jsx-no-useless-fragment */}
```

Comment:
As we've now had to disable this rule in multiple places, is it worth creating a simple wrapper component around a fragment that explains why we are using it like this? That way we on't have to keep disabling `eslint`.

Also, as someone who's not worked closely on the `SchemaForm` before, it's not clear as to why we're doing this. Personally I'd find it useful to have this explained in code.

Reply:
I've removed this altogether and it seems to be working. I think I'll leave it in the other part of the code for now as I don't want to touch the upload functionality in this ticket, just in case something goes wrong!